tools/taskfiles/taskfile_code_quality.yml

```diff
@@ -302,3 +302,23 @@ tasks:
         " > {{.ROOT_DIR}}/.git/hooks/pre-commit
       - chmod +x {{.ROOT_DIR}}/.git/hooks/pre-commit
     silent: true
+
+  find_and_check_requirements:
+    shell: bash
+    vars:
+      TOML_DIRS:
+        sh: find {{.ROOT_DIR}}/app {{.ROOT_DIR}}/lib -name 'pyproject.toml' | awk -F / '{FS=OFS="/";$NF=""; print $0}'
+    cmds:
+      - for: { var: TOML_DIRS, as: TOML_DIR }
+        task: check_missing_requirements
+        vars:
+          TOML_DIR: '{{.TOML_DIR}}'
+
+  check_missing_requirements:
+    shell: bash
+    cmds:
```

Comment:
can we split those into separate cmds?

Reply:
What do u mean? Separate tasks for cd, deptry and result? As far as I know cd won't work

Reply:
You can add a directory for the task to avoid using `cd`: https://taskfile.dev/usage/#task-directory